libcodechecker/server/database/config_db_model.py

```diff
@@ -106,6 +107,22 @@ def __init__(self, permission, product_id, name, is_group=False):
         self.name = name
         self.is_group = is_group
 
+
+class Session(Base):
```

Comment:
Wouldn't we mix the current database sessions with the user login sessions? I'm just thinking about a different naming here.

Reply:
You import them with different names using `from ... import ... as ...`. The user login session, as an object, is an internal thing to `session_manager.py`, and should not be imported by code outside of it.